build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
 
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:3.5.3"
+        classpath 'com.android.tools.build:gradle:3.6.2'
```

Comment:
In general, we might want to make upgrades to build files in separate commits and PRs.